webapi/Controllers/ChatHistoryController.cs

```diff
@@ -240,4 +241,78 @@ public async Task<IActionResult> GetAllChatSessionsAsync(string userId)
 
         return this.NotFound($"No chat session found for chat id '{chatId}'.");
     }
+
+    /// <summary>
+    /// Delete a chat session.
+    /// </summary>
+    /// <param name="requestParameters">Object that contains the parameters to delete the chat.</param>
+    [HttpPost]
+    [Route("chatSession/delete")]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> DeleteChatSessionAsync([FromServices] IHubContext<MessageRelayHub> messageRelayHubContext, [FromBody] DeleteChatRequest requestParameters)
```

Comment:
Once Gina's auth and security changes are in, we'll only need to provide the chat ID, I believe, as the user id will be retrievable from the auth...

Just an FYI for the future.

Reply:
Good call, I'll add a TODO here to handle that